cv32e40p/regress/cv32e40p_debug.yaml

```diff
@@ -35,8 +35,8 @@ tests:
     build: uvmt_cv32e40p
     description: Debug reset test with random boot set
     dir: cv32e40p/sim/uvmt
-    cmd: make test TEST=debug_test_boot_set
-    num: 50
+    cmd: make test TEST=debug_test_boot_set SEED=1 VSIM_USER_FLAGS=+fixed_instr_gnt_stall=10
```

Comment:
Hmmm.  Fixed OBI timing is not a good idea.  Is there a reason for this?

Reply:
@pascalgouedo , Could you help to comment on this. I believed it was added due to the test issue.

Reply:
Only way to make it running fine and make it follow the original intent.
Without it it fails depending of seed and instr OBI latency.
Don't even understand how it could have worked correctly in v1... But it was a different test-bench then.
I have no experience in uvm sequences but for me it can't honour the original intent of having debug request before any instruction execution in decode stage.
Up to you to run it without those seed and obi latency options and make it work as intentionally created together with any random seed 😀.

Reply:
Hmmm.  That issue [#2162](https://github.com/openhwgroup/core-v-verif/issues/2162) definitately has fallen through the cracks - my bad. The issue is still assigned to me, so I will add it back to the top of my ToDo pile.  In the meantime, we can merge this in.